### PR TITLE
cross-compile for aarch64

### LINKED ISSUE #358: Pre-compiled ARM binaries
As a user, I would like to be able to download `aarch64` prebuilt binaries so I can run Krustlet on 64 bit ARM devices (like Raspberry Pi).

Other notes:

We may also want to have an "openssl-less" ARM binary to make it easier to just drop in on an embedded device

### PR DESCRIPTION
**Pull request by bacongobbler**
closes #358

Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -18,6 +18,14 @@ jobs:
               args: "--release",
               extension: "",
             }
+          - {
+              os: "ubuntu-latest",
+              arch: "aarch64",
+              args: "--release --target aarch64-unknown-linux-gnu",
+              extension: ".exe",
```
Comment (thomastaylor312) [inline .github/workflows/release.yml:25]: This extension should be empty

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -57,7 +65,24 @@ jobs:
           rustup component add clippy --toolchain stable-x86_64-apple-darwin
           rustup update stable
 
+      - name: setup for cross-compile builds
+        if: runner.os == 'linux'
```
Comment (thomastaylor312) [inline .github/workflows/release.yml:69]: We should make this `if` statement use something in the matrix config so we don't end up compiling OpenSSL during a normal linux build

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
Excited to have this. Just two small fixes needed

File: .github/workflows/release.yml (hunk 1 above)
Comment (bacongobbler) [reply .github/workflows/release.yml:25]: good catch

File: .github/workflows/release.yml (hunk 2 above)
Comment (bacongobbler) [reply .github/workflows/release.yml:69]: hmm... good idea. perhaps `matrix.config.arch == "aarch64"` would do

**Review by thomastaylor312 — APPROVED** [review]

**Comment by thomastaylor312** [comment]
This is good to go from the code/configuration side. Looks like something broke with the pipeline though

**Comment by bacongobbler** [comment]
all tests passed. Github actions is holding onto the old matrix, but that should fix itself once this is merged.

**Pull request merged**